runelite-client/src/main/java/net/runelite/client/plugins/chompy/ChompyInfoOverlay.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
This should be an infobox instead of a panel, just have hte text as "?" if kc is unknown, with tooltip telling you to check kc